libraries/load-themed-styles/config/jest.config.json

```diff
@@ -1,3 +1,7 @@
 {
-  "extends": "local-web-rig/profiles/library/config/jest.config.json"
+  "extends": "local-web-rig/profiles/library/config/jest.config.json",
+
+  "rootDir": "lib",
+  "roots": ["<rootDir>/lib"],
+  "testMatch": ["<rootDir>/lib/**/*.test.js"]
```

Comment:
What's in the rig?

Reply:
It was web-rig, so expected `lib/` to contain ESM. Now normalized to have `lib/` not be a real folder.